pages/users/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { Typography, Box, TextField, Stack, Button } from "@mui/material";
+import SelectRoleTextField from "../../components/CreateNewUserAccount/SelectRoleTextField";
+import UploadProfileImage from "../../components/CreateNewUserAccount/UploadProfileImage";
+import CreateButton from "../../components/CreateNewUserAccount/CreateButton";
+
+
+const CreateNewUserAccount = () => {
+
+    const [inputValue, setInputValue] = React.useState(null);
+
+    return (
+        <Box>
+            <Typography variant="h4" sx={styledTitle}>Create New User Account</Typography>
+            <Stack spacing={10} sx={styledStack}>
+                <UploadProfileImage />
+                <TextField id="outlined-basic" label="Name of User" variant="outlined" required value={inputValue}/>
+                <TextField id="outlined-basic" label="Email Address" variant="outlined" required/>
```

Comment:
you will need to use the `onChange` prop if you are using it as a controlled component. or a `useRef` if u are using it as an uncontrolled component. Either way is fine just keep it consistent